tests/stubdata/output/jats_liebert_atypon.json

```diff
@@ -0,0 +1,133 @@
+{
+  "authors": [
+    {
+      "affiliation": [
+        {
+          "affPubRaw": "Department of Aeronautics and Astronautics, Stanford University, Stanford, CA."
+        }
+      ],
+      "name": {
+        "given_name": "\n\nModerator:\n Scott\n\t\t\t\t\t\t",
```

Comment:
Doh!  I'm sorry if that was in the json I gave you, that should've been stripped -- if not the "Moderator:" part, then definitely the `\n\t`.